frontend/packages/topology/src/components/list-view/TopologyListViewAppGroup.tsx

```diff
@@ -52,7 +52,7 @@ const TopologyListViewAppGroup: React.FC<TopologyListViewAppGroupProps> = ({
   cells.push(
     <DataListCell
       key={id}
-      className="odc-topology-list-view__application-label-cell"
+      className="odc-topology-list-view__application-label-cell odc-topology-list-view__nofocus-label"
```

Comment:
Don't find any css definition for `odc-topology-list-view__nofocus-label` ?! Can we drop this again?

```suggestion
      className="odc-topology-list-view__application-label-cell"
```

Reply:
updated